tools/topology/sof-tgl-max98357a-rt5682.m4

```diff
@@ -0,0 +1,202 @@
+#
+# Topology for Tigerlake with Max98357a amp + rt5682 codec + DMIC + 4 HDMI
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`ssp.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Tigerlake DSP configuration
+include(`platform/intel/tgl.m4')
+include(`platform/intel/dmic.m4')
+DEBUG_START
+
+#
+# Define the pipelines
+#
+# PCM0 <---> volume <----> SSP1  (Speaker - max98357a)
```

Comment:
this seems to be playback only, not duplex, so arrows one direction, please

Reply:
fixed and updated patch now.